google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpcViews.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner.spi.v1;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import io.opencensus.stats.*;
+import io.opencensus.stats.Aggregation.Distribution;
+import io.opencensus.stats.Aggregation.Sum;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.tags.TagKey;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@VisibleForTesting
+public class SpannerRpcViews {
+
+  /** Unit to represent milliseconds. */
+  private static final String MILLISECOND = "ms";
+  /** Unit to represent counts. */
+  private static final String COUNT = "1";
+
+  /** TagKeys */
+  public static final TagKey METHOD = TagKey.create("method");
+
+  public static final TagKey PROJECT_ID = TagKey.create("project_id");
+  public static final TagKey INSTANCE_ID = TagKey.create("instance_id");
+  public static final TagKey DATABASE_ID = TagKey.create("database");
```

Comment:
Just checking if we are using `database` and not `database_id` for other metrics as well